### ISSUE TITLE
contrib/media.py: Exeption "Cannot flush AudioResampler before it is used" 

### ISSUE THREAD

**Issue by andrews239** [aiortc/aiortc#259]
AV version 6.2 prohibits flushing AudioResampler before receiving the first significant frame

```
Traceback (most recent call last):
  File "a.py", line 563, in <module>
    loop.run_until_complete(main())
  File "/usr/lib/python3.8/asyncio/base_events.py", line 608, in run_until_complete
    return future.result()
  File "a.py", line 548, in main
    await  APP["Items"][hqone_Space].data[rec].stop()
  File "~/.local/lib/python3.8/site-packages/aiortc/contrib/media.py", line 361, in stop
    for packet in context.stream.encode(None):
  File "av/stream.pyx", line 130, in av.stream.Stream.encode
  File "av/codec/context.pyx", line 311, in av.codec.context.CodecContext.encode
  File "av/audio/codeccontext.pyx", line 40, in av.audio.codeccontext.AudioCodecContext._prepare_frames_for_encode
  File "av/audio/resampler.pyx", line 58, in av.audio.resampler.AudioResampler.resample
ValueError: Cannot flush AudioResampler before it is used.
```

**Comment by andrews239** [comment]
See also: https://github.com/mikeboers/PyAV/issues/537 (Fixed but not released)

**Comment by andrews239** [comment]
```
--- media.py.orig<----->2019-12-13 18:31:25.408617734 -0800
+++ media.py<-->2020-02-03 23:47:29.642706337 -0800
@@ -358,8 +358,12 @@
                 if context.task is not None:
                     context.task.cancel()
                     context.task = None
-                    for packet in context.stream.encode(None):
-                        self.__container.mux(packet)
+                    try:
+                        for packet in context.stream.encode(None):
+                            self.__container.mux(packet)
+                    except ValueError:
+                        print( "ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZzz")
+                        pass
             self.__tracks = {}
 
             if self.__container:
```

**Comment by jlaine** [comment]
The PyAV issue you mentioned was fixed and released in version 7.0.0.
